### ISSUE TITLE
Processing for empty rdg elements

### ISSUE THREAD

**Issue by JanelleJenstad** [projectLEMDO/lemdoIssues#35]
The TEI Guidelines 12.4 ([https://tei-c.org/release/doc/tei-p5-doc/en/html/TC.html#index-body.1_div.12_div.4](https://tei-c.org/release/doc/tei-p5-doc/en/html/TC.html#index-body.1_div.12_div.4)](https://tei-c.org/release/doc/tei-p5-doc/en/html/TC.html#index-body.1_div.12_div.4) indicate that when a witness omits text, the collation should register the omission with an empty `<rdg>` element. 

What is our processing in that case? Can we dynamically add in `Omitted` after the siglum? 

I'm asking because I think that no one has actually properly used an empty `<rdg/>` element before now. Editors have typed "Omitted" or "Not in Q1" or something like that in the text node. 

The file in which I've just creating an empty `<rdg/>` element is emdR2_M_collation.xml

**Comment by martindholmes** [comment]
@JanelleJenstad I think this is already handled as it should be -- here's the template:

~~~~~
  <xd:doc>
    <xd:desc>Handle readings similarly to lemmas, but we have to check
    whether or not it is omitted, which is implied by a lack of content.</xd:desc>
  </xd:doc>
  <xsl:template match="app/rdg" mode="tei">
    <div>
      <xsl:call-template name="processAtts"/>
      <div>
        <!--If there's no descendant text outside of the context of a note, then it's
          presumably empty-->
        <xsl:if test="normalize-space(string-join(descendant::text()[not(ancestor::note)],'')) = ''">
          <div class="omit"/>
        </xsl:if>
        <!--But even if it is omitted, there still might be a note to process-->
        <xsl:apply-templates mode="#current"/>
      </div>
    </div>
  </xsl:template>
~~~~~

In other words, an empty div with the class "omit" is created. This then USED to be handled in the CSS using generated content, showing "(Omitted)". However, a rewrite of the CSS at some point removed all of that. Pat has now put that back into the current CSS, so it should show up in the next build after he does a commit some time today.

**Comment by JanelleJenstad** [comment]
Great! Thanks so much for looking into that and having Pat put back the original CSS. I've documented the practice of using empty <rdg/> elements in learn_encodeCollations just now. 
Undoubtedly, I will now have to go look for instances of editors typing Om. and Omitted in their collations, but that is a task for another list. 

I will close this ticket now, with thanks.
